js/samples/docs-menu-generation/src/index.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This sample is referenced by the genkit docs. Changes should be made to
+// both.
+import { generate, generateStream } from '@genkit-ai/ai';
+import { configureGenkit } from '@genkit-ai/core';
+import { defineFlow, startFlowsServer } from '@genkit-ai/flow';
+import { geminiPro, geminiProVision, googleAI } from '@genkit-ai/googleai';
+import * as z from 'zod';
+
+configureGenkit({
+  plugins: [googleAI()],
+  logLevel: 'debug',
+  enableTracingAndMetrics: true,
+});
+
+export const menuStreamingSuggestionFlow = defineFlow(
```

Comment:
kind of a nit, but its weird to split the name like this: `menu<Streaming>SuggestionFlow`. alternatives:

- `streamingMenuSuggestionFlow`.
- `menuSuggestionFlowStreaming`.
- `menuSuggestionStreamingFlow`.